README.rst

```diff
@@ -102,6 +101,13 @@ In e.g. your ``init.lua``:
                 -- this if so desired.
                 wknight = '♘',
             },
+            -- change if you don't like the backslash
+            -- (comma is a popular choice on French keyboards)
+            leader = '\\',
+            -- Set one of the following to true to enable abbreviations
+            compe = false, -- nvim-compe source
```

Comment:
Maybe reorder these to the top of the table here, since without setting them none of the rest matters much.